Font_Awesome_Cookie_Info.md

```diff
@@ -6,49 +6,49 @@ Publicly available
 
 ## Review Information
 
-__Mandatory Review Period__
+**Mandatory Review Period**
```

Comment:
Why this change?  Is it semantically different somehow?  I don't see a visual change in the display in GitHub.